### PR TITLE
Feature/withdraw invitation

### PR DESCRIPTION
**Pull request by mohituniyal**
Allows authors to withdraw their submissions.
Uses class SubmissionStage's attributes "allow_withdraw" and "reveal_authors_on_withdraw" while enabling withdraw invitations.
Creates 1 conference level CONF_ID+"Withdrawn_Submissions" invitation and then a per paper invitation names CONF_ID+"Paper<paper-number>/Withdraw".

### CODE REVIEW

File: openreview/conference/builder.py [hunk 1]
```diff
@@ -819,6 +835,10 @@ def get_blind_submission_id(self, conference):
             name = 'Blind_' + name
         return conference.get_invitation_id(name)
 
+    def get_withdrawn_submission_id(self, conference):
+        name = 'Withdrawn_Submission'
```
Comment (mspector) [inline openreview/conference/builder.py]: you could make this a default parameter:

`def get_withdrawn_submission_id(self, conference, name='Withdrawn_Submission'):`

File: openreview/conference/invitation.py [hunk 2]
```diff
@@ -274,6 +272,102 @@ def __init__(self, conference):
...
+        if (conference.submission_stage.double_blind and not conference.submission_stage.reveal_authors_on_withdraw):
+            content['authors'] = {
+                'values': ['Anonymous']
+            }
+            content['authorids'] = {
+                'values-regex': '.*'
+            }
+
+        super(WithdrawnSubmissionInvitation, self).__init__(
+            id=conference.get_invitation_id('Withdrawn_Submission'),
```
Comment (mspector) [inline openreview/conference/invitation.py]: maybe this should be `conference.submission_stage.get_withdrawn_submission_id()`?

Comment (mohituniyal) [reply openreview/conference/invitation.py]: Agreed. I have made the changes.

File: openreview/conference/builder.py (hunk 1 above)
Comment (mohituniyal) [reply openreview/conference/builder.py]: Agreed.

**Pull request merged**